coral/src/app/features/browse-topics/BrowseTopics.test.tsx

```diff
@@ -12,6 +12,7 @@ import userEvent from "@testing-library/user-event";
 import { mockGetEnvironments } from "src/domain/environment";
 import { mockedTeamResponse, mockGetTeams } from "src/domain/team/team-api.msw";
 import { mockedEnvironmentResponse } from "src/domain/environment/environment-api.msw";
+import { mockIntersectionObserver } from "src/services/test-utils/mock-intersection-observer";
```

Comment:
Not related to this line, but I think we are missing test cases for the "search topics by name" flow in this file.

Reply:
😱  😱  😱  😱  😱  😱  😱   thanks for noticing! 
I'll add a follow up ticket. [#29](https://github.com/aiven/klaw/issues/292)